.github/workflows/cibuild.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         run: |
           mkdir ~/.taxonkit
           pushd ~/.taxonkit
-          curl -L -O ftp://ftp.ncbi.nih.gov/pub/taxonomy/taxdump.tar.gz
+          curl -L -O https://ftp.ncbi.nih.gov/pub/taxonomy/taxdump.tar.gz
```

Comment:
The MacOS build kept on failing, unable to download the taxonomy DB. This was an attempt at troubleshooting.